README.md

```diff
@@ -192,6 +192,11 @@ For sync communication we have a community slack with a #containerd channel that
 
 **Slack:** https://dockr.ly/community
 
+### Security Mailing List
+
+__If you are reporting a security issue, please follow the responsible
+disclosure guidelines and reach out discreetly at containerd-security@googlegroups.com__.
```

Comment:
Which guidelines? 

http://www.cert.org/vulnerability-analysis/vul-disclosure.cfm ?

Reply:
Not sure, I copied this from the RELEASE doc

Reply:
where's @stevvooe when you need him :trollface: